app/app.rb

```diff
@@ -3,11 +3,21 @@
 ENV['RACK_ENV'] ||= 'development'
 
 require 'sinatra/base'
+require 'sinatra/flash'
 require './database_setup.rb'
 
 class DwellBNB < Sinatra::Base
 
   enable :sessions
+  set :session_secret, '7a929c0c04b166475878f813dd25d869b5ad4e631eed3194edb2555e22039a94'
```

Comment:
Could we not hide this secret as an ENV variable?

Reply:
But otherwise :shipit:

Reply:
At some point I think we could